cves/CVE-2013-0863.yml

```diff
@@ -199,8 +206,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: true
+  answer: The implicit requirement that the system should be functioning at all times. A system crash
```

Comment:
Won't comment on them all but I think you're missing a few `|` for multi-line comments. I'd look for these if you're having any build issues.